tools/buildroot_config

```diff
@@ -23,3 +23,4 @@ BR2_PACKAGE_UTIL_LINUX=y
 BR2_PACKAGE_UTIL_LINUX_BINARIES=y
 BR2_PACKAGE_PYTHON3=y
 # BR2_TARGET_ROOTFS_TAR is not set
+BR2_PACKAGE_BUSYBOX_CONFIG_FRAGMENT_FILES="../../tools/buildroot_busybox_fragments"
```

Comment:
Where does the relative path come from ? Is it relative to where BusyBox gets built ?

Reply:
Yes, it's relative to $LISA/src/buildroot . I figured it's best to put buildroot_busybox_fragments under $LISA/tools next to buildroot_config .